astropy/modeling/core.py

```diff
@@ -3763,7 +3763,12 @@ def _custom_model_wrapper(func, fit_deriv=None):
 
     members.update(params)
 
-    return type(model_name, (FittableModel,), members)
+    cls = type(model_name, (FittableModel,), members)
+    if len(inputs) == 1:
+        cls._separable = True
+    else:
+        cls._separable = False
```

Comment:
```suggestion
    cls._separable = True if (len(inputs) == 1) else False
```

Reply:
Ooh, I like this version much more!

Reply:
But you just reverted it again?

Reply:
Ah my bad, I rebased and force pushed it away, will fix back to the ternary.